src/cli.rs

```diff
@@ -215,10 +247,30 @@ fn get_src_bin_tree_sitter_lint_local_rs_contents(local_binary_crate_name: &str)
 fn get_src_bin_tree_sitter_lint_local_lsp_rs_contents(local_binary_crate_name: &str) -> String {
     let local_binary_crate_name = format_ident!("{}", local_binary_crate_name);
     quote! {
-        use tree_sitter_lint::tokio;
+        use std::env;
+
+        use tracing_chrome::ChromeLayerBuilder;
+        use tracing_subscriber::{prelude::*, EnvFilter};
+        use tree_sitter_lint::{tokio, squalid::NonEmpty};
 
         #[tokio::main]
         async fn main() {
+            if env::var("TRACE_CHROME").ok().is_non_empty() {
+                let (chrome_layer, _guard) = ChromeLayerBuilder::new().include_args(true).file("/Users/jrosse/prj/hello-world/trace.json").build();
+                tracing_subscriber::registry().with(chrome_layer).init();
```

Comment:
I think I hadn't gotten as far as figuring out how to get it to actually spit out tracing data (for "chrome" profiling) when running as the LSP server (since that doesn't just "end" cleanly at a known time) but I think maybe I'd found an API in tracing-chrome that would let me manually trigger dumping tracing data and certain times?